example/object_detection_ssd_mobilenet/scripts/download_model.sh

```diff
@@ -0,0 +1,13 @@
+# Delete old models
+rm -r assets/models/
+
+# Create folder if not already exists
+mkdir assets/models
```

Comment:
So not having assets/models initially doesn't actually work with our CI. Maybe we can create the directory up front and put a README in each so it exist in the repo, then still have the scripts still for downloading the models.

Reply:
But what should be ignored then? We of course put `.tflite` in the .gitignore but should we also add `.txt`? 
Also, in the text_classification example, when we unzip the model it just returns `vocab` as a file without ending. 
Should i also just add this to `example/.gitignore`?

Reply:
Closing this out for now - I _think_ CI might have fixed itself from another run, so rerunning it now to verify